zerver/lib/actions.py

```diff
@@ -1644,7 +1651,7 @@ def create_stream_if_needed(realm: Realm,
     if created:
         Recipient.objects.create(type_id=stream.id, type=Recipient.STREAM)
         if stream.is_public():
-            send_stream_creation_event(stream, active_user_ids(stream.realm_id))
+            send_stream_creation_event(stream, list(public_stream_user_ids(stream)))
```

Comment:
Why isn't this `active_non_guest_user_ids`?  There will be no subscribers at this point in the stream creation process.

Reply:
Yep, fixed.